prowler/providers/aws/services/fms/fms_service.py

```diff
@@ -0,0 +1,109 @@
+from botocore.client import ClientError
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.aws_provider import generate_regional_clients
+
+
+################## FMS
+class FMS:
+    def __init__(self, audit_info):
+        self.service = "fms"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audited_partition = audit_info.audited_partition
+        self.audit_resources = audit_info.audit_resources
+        global_client = generate_regional_clients(
+            self.service, audit_info, global_service=True
+        )
+        self.client = list(global_client.values())[0]
+        self.region = self.client.region
+        self.fms_admin_account = True
+        self.fms_policies = []
+        self.__list_policies__()
+        self.__list_compliance_status__()
+
+    def __list_policies__(self):
+        logger.info("FMS - Listing Policies...")
+        try:
+            list_policies_paginator = self.client.get_paginator("list_policies")
+            try:
+                for page in list_policies_paginator.paginate():
+                    for fms_policy in page["PolicyList"]:
+                        if not self.audit_resources or (
+                            is_resource_filtered(
+                                fms_policy["PolicyArn"], self.audit_resources
+                            )
+                        ):
+                            self.fms_policies.append(
+                                FMSPolicy(
+                                    arn=fms_policy.get("PolicyArn"),
+                                    id=fms_policy.get("PolicyId"),
+                                    name=fms_policy.get("PolicyName"),
+                                    resource_type=fms_policy.get("ResourceType"),
+                                    service_type=fms_policy.get("SecurityServiceType"),
+                                    remediation_enabled=fms_policy.get(
+                                        "RemediationEnabled"
+                                    ),
+                                    delete_unused_managed_resources=fms_policy.get(
+                                        "DeleteUnusedFMManagedResources"
+                                    ),
+                                )
+                            )
+            except ClientError as error:
+                if error.response["Error"]["Code"] == "AccessDeniedException":
+                    print(error.response)
```

Comment:
Is this `print` necessary ? 
Also are we sure that the `"AccessDeniedException"` is retrieved only when there is no default admin?
May be we should parse the body of the error and not only the code to ensure that we are detecting what we want